src/BlazorWasmSdk/Tasks/Microsoft.NET.Sdk.BlazorWebAssembly.Tasks.csproj

```diff
@@ -1,35 +1,54 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
 
   <PropertyGroup>
     <BlazorWasmSdkRoot>$(RepoRoot)\src\BlazorWasmSdk\</BlazorWasmSdkRoot>
     <PackageId>Microsoft.NET.Sdk.BlazorWebAssembly</PackageId>
     <OutDirName>$(Configuration)\Sdks\$(PackageId)\tools</OutDirName>
   </PropertyGroup>
 
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
   <PropertyGroup>
-    <Description>SDK for building and publishing Blazor WebAssembly applications.</Description>
     <OutputType>Library</OutputType>
-    <RootNamespace>Microsoft.NET.Sdk.BlazorWebAssembly</RootNamespace>
-    <TargetFrameworks>$(SdkTargetFramework);net472</TargetFrameworks>
-  </PropertyGroup>
 
-  <PropertyGroup>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <EnableDefaultItems>false</EnableDefaultItems>
     <GenerateDependencyFile>false</GenerateDependencyFile>
     <IncludeBuildOutput>false</IncludeBuildOutput>
 
     <NoPackageAnalysis>true</NoPackageAnalysis>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <!-- MSBuild Task DLLs need to be versioned with every build -->
     <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
-  </PropertyGroup>
 
-  <PropertyGroup>
     <OutputPath>$(BaseOutputPath)</OutputPath>
     <IsPackable>true</IsPackable>
     <PackageLayoutOutputPath>$(ArtifactsBinDir)$(Configuration)\Sdks\$(PackageId)\</PackageLayoutOutputPath>
+    <NoDefaultExcludes>true</NoDefaultExcludes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>SDK for building and publishing Blazor WebAssembly applications.</Description>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.NET.Sdk.BlazorWebAssembly</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TargetFrameworks>$(SdkTargetFramework);net46</TargetFrameworks>
```

Comment:
Is there a reason this changed from `net472` -> `net46`?

Reply:
It matches the target framework in the [original SDK](https://github.com/dotnet/aspnetcore/blob/f7e1712db88db4c01fcc49e5995d9528dbf608df/src/Components/WebAssembly/Sdk/src/Microsoft.NET.Sdk.BlazorWebAssembly.csproj#L4).

Reply:
Could we change this to `net472` if it lets us use `MicrosoftBuildUtilitiesCoreVersion`?

Reply:
Tried to get this working. Changing the target to net472 breaks the full framework tests which target net48 by default. I updated the target framework for the tests [here](https://github.com/dotnet/sdk/pull/14968/commits/52f0752688c01b8914038622f3f5f6b97933db99) but the end result is that it tries to build for both net472 and .NET Core on all Windows tests, including the ones that don't validate the full framework.

@wli3 Is there a way to configure the framework to target net472 and only build the tests for net472 under the full framework tests on Windows and not the other builds on Windows?

Reply:
not other full frameworks like net48 or not on .net core?

Reply:
Correct. Assuming you don't explicitly set the TargetFramework in the Test project, the tests will be built to target .NET Core. In the full framework scenario, the tests will be built to target .NET Core, but the underlying MSBuild will validate on .NET Framework (at least that's what I've assessed from the current test run). Is there a way to build the tests to target .NET Core, but to use a particular framework version in the evaluation?

Reply:
If building with net46 requires pinning the tasks versions, but does not have any other side-effects (all the tests appear to be passing), can we check that in and file follow up issue to clean this up? I wouldn't want this PR to languish for several weeks because of this. Particularly because it blocks VS \ James from doing their gRPC work. (FYI @mkArtakMSFT)

Reply:
Yes, we can do that. As long as the tests don't fail, we should be good to merge this and follow-up later.